lldb/test/API/functionalities/gdb_remote_client/TestWasm.py

```diff
@@ -89,6 +91,9 @@ def readMemory(self, addr, length):
             file.close()
         return result
 
+    def qWasmCallStack(self, data):
+        return "b301000000000040fe01000000000040"
```

Comment:
Comment to explain data.

I would guess 2 64 bit address values, one for each thread?